src/lib/resources.ts

```diff
@@ -159,6 +159,26 @@ export class Items extends OPResource {
         );
     }
 
+    /**
+     * Search for all Items with exact match on title.
```

Comment:
Tiny nitpick (to match other doc formatting):
```suggestion
     * Search for all Items with exact match on Title.
```

Reply:
fixed